schema/rdl_schema_0.1.json

```
@@ -36,7 +36,7 @@
     "risk_data_type": {
       "title": "Risk data type",
       "type": "string",
-      "description": "The type of risk data included in the dataset, from the closed [risk data type codelist](https://rdl-standard.readthedocs.io/en/{{version}}/reference/codelists/#risk-data-type).",
+      "description": "The type of risk data included in the dataset, from the closed [risk data type codelist](https://rdl-standard.readthedocs.io/en/{{version}}/references/codelists/#risk-data-type).",
```

Not sure why all the codelist links have been updated, the old links (without the trailing 's' on reference) were correct.

```suggestion
      "description": "The type of risk data included in the dataset, from the closed [risk data type codelist](https://rdl-standard.readthedocs.io/en/{{version}}/reference/codelists/#risk-data-type).",
```

---

that was my mistake!